build.sbt

```
@@ -14,6 +14,12 @@ ThisBuild / developers := List(
 ThisBuild / mergifyStewardConfig ~= { _.map(_.copy(mergeMinors = true)) }
 ThisBuild / mergifySuccessConditions += MergifyCondition.Custom("#approved-reviews-by>=1")
 ThisBuild / mergifyLabelPaths += { "docs" -> file("docs") }
+ThisBuild / mergifyPrRules += MergifyPrRule("assign scala-steward's PRs for review",
+  List(MergifyCondition.Custom("author=scala-steward")),
```

While you're here :)

```suggestion
  List(MergifyCondition.Custom("author=typelevel-steward[bot]")),
```